src/targets/happymodel_900.ini

```diff
@@ -32,6 +32,7 @@ build_flags =
 	${env_common_esp32.build_flags}
 	${common_env_data.build_flags_tx}
 	-D TARGET_ES900TX
+	-D WS2812_IS_GRB
```

Comment:
Does the HM900 module module have a different LED to the HM2400?

Reply:
I do not know, all I know is that my ES900TX is GRB. @AlessandroAU do you have a HM 2400 with an RGB LED?